helm-charts/verticadb-operator/README.md

```diff
@@ -27,6 +27,7 @@ This helm chart will install the operator and an admission controller webhook.
 | reconcileConcurrency.verticadb | Set this to control the concurrency of reconciliations of VerticaDB CRs | 5 |
 | reconcileConcurrency.verticarestorepointsquery | Set this to control the concurrency of reconciliations of VerticaRestorePointsQuery CRs | 1 |
 | reconcileConcurrency.verticascrutinize | Set this to control the concurrency of reconciliations of VerticaScrutinize CRs | 1 |
+| reconcileConcurrency.sandboxconfigmap | Set this to control the concurrency of reconciliations of ConfigMaps that contain state for a sandbox  | 1 |
```

Comment:
The parms are sorted alphabetically. So, it needs to move up right after `reconcileConcurrency.eventtrigger`